docs/content/users/basics/architecture.md

```diff
@@ -0,0 +1,109 @@
+# How DDEV Works
+
+DDEV is a [Go](https://go.dev) application that stores its configuration in [files on your workstation](#directory-tour).  
+It uses those blueprints to mount your project files into [Docker containers](#container-architecture) that facilitate the operation of a local development environment.
+
+DDEV writes and uses [docker-compose](https://docs.docker.com/compose/) files for you, which is a detail you can cheerfully ignore unless you’re Docker-curious or [defining your own services](../extend/custom-compose-files.md).
+
+## Directory Tour
+
+A project’s `.ddev` directory can be intimidating at first, so let’s take a look at what lives in there.
```

Comment:
Should we make a distinction here between project .ddev and global .ddev? This is clear enough, but people do stumble on that sometimes. Maybe add another tip? Say that the project directory and global have different purposes and global is in homedir?

Reply:
I didn’t even consider that, but yes! It seems vital to make room for this, not in a tip but expanding the page since DDEV’s global directory is equally important even if you don’t see it as much. Let me incorporate your other suggestions and come back to this.